aiida/cmdline/commands/cmd_archive.py

```diff
@@ -254,6 +254,7 @@ def migrate(input_file, output_file, force, in_place, archive_format, version, v
 
 class ExtrasImportCode(Enum):
     """Exit codes for the verdi command line."""
+    # pylint: disable=invalid-name
```

Comment:
Won't this now disable this for the entire file? Or is it properly scoped to the `enum`?

Reply:
yeh, I was hopping it would be scoped to the `enum`, or do you think not?

Reply:
Not sure to be honest. You could give it a quick try by adding a global constant after the enum declaration which should then incur a warning from pylint if it is properly scoped.

Reply:
Yep looks to work as intended, FYI scoping is detailed here: http://pylint.pycqa.org/en/latest/user_guide/message-control.html#block-disables